tests/db_utils.py

```diff
@@ -87,3 +91,33 @@ def insert_record(conn_cursor, table_name, data):
                             ( {} )
                      VALUES ( {} )""".format(quote_ident(table_name, conn_cursor), columns_sql, value_sql)
     conn_cursor.execute(insert_sql, our_values)
+
+
+def update_record(conn_cursor, ctable_name, primary_key, data):
+    """
+    Update an existing record as specified using the following params.
+    :param conn_cursor:    A pyschopg2 connection object.
+    :param ctable_name:    The canonicalized talbe name.
+    :param primary_key:    The value of the primary key
+                           of the record you want to update.
+    :param data:           A dictionary of fields to values to
+                           update in the record.
+    """
+    fields_to_update = ""
+    for field, value in data.items():
+        if ' ' in field:
+            field = quote_ident(field, conn_cursor)
+        fields_to_update += " {} = '{}',".format(field, value)
+
+    update_sql = "UPDATE {} SET{} WHERE id = {}".format(ctable_name,
+                                                        fields_to_update[:-1],
```

Comment:
isn't this dropping the last field to update?

Reply:
Since `fields_to_update` is a string, we are only dropping the last character which is a extra comma that gets added due to the way we build the string in the for loop above.

Reply:
acha.